Gemfile.lock

```diff
@@ -441,6 +443,7 @@ DEPENDENCIES
   devise
   dotenv-rails
   email_address
+  fast_jsonapi
```

Comment:
Did this creep back in even after removing the `example_serializer`?

Reply:
Gone now :)